test/app_manifest/addon_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+module AppManifest
+  class AddonTest < MiniTest::Test
+    def test_attributes
```

Comment:
I feel like test cases that only verify that Virtus attributes work the way Virtus says they do might be excessive.